### ISSUE TITLE
"AttributeError: 'Table' object has no attribute 'count'" if sqlalchemy >=1.4

### ISSUE THREAD

**Issue by zhangchunlin** [limodou/uliweb3#70]
How to reproduce method:
- pip install sqlalchemy with version >=1.4
- nosetests --with-doc test_orm:test_1

https://docs.sqlalchemy.org/en/13/core/selectable.html?highlight=fromclause%20count#sqlalchemy.sql.expression.FromClause.count

Deprecated since version 1.1: The [FromClause.count()](https://docs.sqlalchemy.org/en/13/core/selectable.html?highlight=fromclause%20count#sqlalchemy.sql.expression.FromClause.count) method is deprecated, and will be removed in a future release. Please use the [count](https://docs.sqlalchemy.org/en/13/core/functions.html#sqlalchemy.sql.functions.count) [func](https://docs.sqlalchemy.org/en/13/core/sqlelement.html#sqlalchemy.sql.expression.Over.func)tion available from the func namespace.

**Comment by zhangchunlin** [comment]
@limodou I try this but didn't work
```
diff --git a/uliweb/orm/__init__.py b/uliweb/orm/__init__.py
index 2d255f6..9e47a3d 100644
--- a/uliweb/orm/__init__.py
+++ b/uliweb/orm/__init__.py
@@ -4700,7 +4700,7 @@ class Model(with_metaclass(ModelMetaclass)):
             
     @classmethod
     def count(cls, condition=None, **kwargs):
-        count = do_(cls.table.count(condition, **kwargs), cls.get_session()).scalar()
+        count = do_(func.count(condition, **kwargs), cls.get_session()).scalar()
         return count
     
     @classmethod
```

**Comment by zhangchunlin** [comment]
After change like this the test_orm:test_1 will pass, but may still have problem:
```
$ git diff
diff --git a/uliweb/orm/__init__.py b/uliweb/orm/__init__.py
index 2d255f6..f705159 100644
--- a/uliweb/orm/__init__.py
+++ b/uliweb/orm/__init__.py
@@ -4700,7 +4700,7 @@ class Model(with_metaclass(ModelMetaclass)):
             
     @classmethod
     def count(cls, condition=None, **kwargs):
-        count = do_(cls.table.count(condition, **kwargs), cls.get_session()).scalar()
+        count = do_(select([func.count()]).select_from(cls.table), cls.get_session()).scalar()
         return count
     
     @classmethod
```

**Comment by limodou** [comment]
change to `count = cls.filter(condition, **kwargs).count()` fixed with f4cb896
